image-rs/src/resource/kbs/mod.rs

```diff
@@ -38,49 +34,61 @@ mod ttrpc_proto;
 mod native;
 
 /// Default workdir to store downloaded kbs resources
-const STORAGE_PATH: &str = "/run/image-security/kbs/";
+const STORAGE_PATH: &str = "image-security/kbs";
 
 /// SecureChannel to connect with KBS
 pub struct SecureChannel {
     /// Get Resource Service client.
     client: Box<dyn Client>,
 
     /// The path to store downloaded kbs resources
-    pub storage_path: String,
+    pub storage_path: PathBuf,
+}
+
+impl Default for SecureChannel {
+    fn default() -> Self {
+        Self::new(
+            "sample_kbc",
+            "null",
+            Path::new(crate::config::DEFAULT_WORK_DIR),
+        )
+        .expect("initialize secure channel")
```

Comment:
nit: add "sample" or "default" to this message to make it slightly easier to find

Reply:
Fixed, thanks!